src/p5xr/core/p5xr.js

```diff
@@ -197,9 +204,8 @@ export default class p5xr {
         // var callMethod = function(f) {
         //   f.call(context);
         // };
-        // TODO Just call a different function that does this minus matrix reset
         if (context._renderer.isP3D) {
-          context._renderer._update();
+          this._updatexr();
```

Comment:
might be safer to use `self._updatexr();` here.

The role of self will need to be reevaluated at some point.